src/main/webapp/app/exercises/programming/manage/programming-exercise-detail.component.html

```diff
@@ -35,6 +35,12 @@ <h2><span jhiTranslate="artemisApp.programmingExercise.detail.title">Programming
                 </button>
             </div>
             <dl class="row-md jh-entity-details">
+                <dt><span jhiTranslate="artemisApp.exercise.course">Course</span></dt>
```

Comment:
May I ask what is preventing you from using your `ExerciseDetailsComponent` here as well?

Reply:
A few things: The programming exercise details has a few attributes which are either missing in the super exercise details (e.g. `buildAndTestStudentSubmissionsAfterDueDate`) and as well has a short name, which should imo directly follow the title. As well the problem statement and grading instructions are formatted differently.

Reply:
Just added a commit, which tries to reuse as much of the component in that exercise detail page, too. Let me know what you think.